src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -28,23 +28,48 @@ public class ApplicationInsightsLoggerOptions : IOptionsFormatter
         public SnapshotCollectorConfiguration SnapshotConfiguration { get; set; }
 
         /// <summary>
+        /// [Deprecated] Please use LiveMetricsAuthenticationApiKey.
         /// Gets or sets authentication key for Quick Pulse (Live Metrics).
         /// </summary>
         public string QuickPulseAuthenticationApiKey { get; set; }
 
         /// <summary>
+        /// Gets or sets authentication key for Live Metrics.
+        /// </summary>
+        public string LiveMetricsAuthenticationApiKey { get; set; }
+
+        /// <summary>
+        /// [Deprecated] Please use LiveMetricsInitializationDelay.
         /// Gets or sets the time to delay Quick Pulse (Live Metrics) initialization. Delaying this initialization
         /// can result in decreased application startup time. Default value is 15 seconds.
         /// </summary>
         public TimeSpan QuickPulseInitializationDelay { get; set; } = TimeSpan.FromSeconds(15);
```

Comment:
Let's make this getter and setter just use LiveMetricsInitializationDelay directly -- so there's only one source of truth until we delete it.

Something like: 
`get { return LiveMetricsInitializationDelay; } set { LiveMetricsInitializationDelay = value; }`

Same goes for the auth api key...